src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,463 @@
+
+### Largest Contentful Paint (LCP) {: #lcp }
+
+[Largest Contentful Paint (LCP)](https://www.web.dev/lcp/) is a measurement of perceived loading
+experience. It marks the point during page load when the primary–or "largest"–content has loaded and
+is visible to the user. LCP is an important complement to First Contentful Paint (FCP) which only
+captures the very beginning of the loading experience. LCP provides a signal to developers about how
+quickly a user is actually able to see the content of a page. An LCP score below 2.5 seconds is
+considered 'Good.'
+
+For more information, watch this deep-dive on LCP by Paul Irish:
+
+{% YouTube 'diAc65p15ag' %}
+
+### Cumulative Layout Shift (CLS) {: #cls }
+
+[Cumulative Layout Shift (CLS)](https://www.web.dev/cls/) is a measurement of visual stability. It
+quantifies how much a page's content visually shifts around. A low CLS score is a signal to
+developers that their users aren't experiencing undue content shifts; a CLS score below 0.10 is
+considered 'Good.'
+
+CLS in a lab environment is measured through the end of page load. Whereas in the field, you can
+measure CLS up to the first user interaction or including all user input.
+
+For more information, watch this deep-dive on CLS by Annie Sullivan:
+
+{% YouTube 'zIJuY-JCjqw' %}
+
+### Total Blocking Time (TBT) {: #tbt }
+
+[Total Blocking Time (TBT)](https://www.web.dev/tbt/) quantifies load responsiveness, measuring the
+total amount of time when the main thread was blocked long enough to prevent input responsiveness.
+TBT measures the total amount of time between First Contentful Paint (FCP) and Time to Interactive
+(TTI). It is a companion metric to TTI and it brings more nuance to quantifying main thread activity
+that blocks a user's ability to interact with your page.
+
+Additionally, TBT correlates well with the field metric [First Input Delay](https://web.dev/fid/)
+(FID), which is a Core Web Vital.
+
+## Performance score update  {: #score }
+
+The [performance score in Lighthouse](https://web.dev/performance-scoring/) is calculated from a
+weighted blend of multiple metrics to summarize a page's speed. The 6.0 performance score formula
+follows.
+
+<table>
+<thead>
+<tr>
+<th><strong>Phase</strong></th>
+<th><strong>Metric Name</strong></th>
+<th><strong>Metric Weight</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Early (15%)</td>
+<td>First Contentful Paint (FCP)</td>
+<td>15%</td>
+</tr>
+<tr>
+<td>Mid (40%)</td>
+<td>Speed Index (SI)</td>
+<td>15%</td>
+</tr>
+<tr>
+<td></td>
+<td>Largest Contentful Paint (LCP)</td>
+<td>25%</td>
+</tr>
+<tr>
+<td>Late (15%)</td>
+<td>Time To Interactive (TTI)</td>
+<td>15%</td>
+</tr>
+<tr>
+<td>Main Thread (25%)</td>
+<td>Total Blocking Time (TBT)</td>
+<td>25%</td>
+</tr>
+<tr>
+<td>Predictability (5%)</td>
+<td>Cumulative Layout Shift (CLS)</td>
+<td>5%</td>
+</tr>
+</tbody>
+</table>
+
+While three new metrics have been added, three old ones have been removed: First Meaningful Paint,
```

Comment:
It does feel like we need a graphic here. Shall we do the 5->6 graphic we did in keynote? I can work with @egsweeny to do this.

Reply:
Is this just the TODO already marked below?

Reply:
![lh-weighting-change-v5-v6](https://user-images.githubusercontent.com/39191/82247717-b8088200-98fb-11ea-8c1b-53ebc3d54d60.png)

This could be better, but we'll use this for now.